src/routes/user-vocabulary/UserVocabularyEditScreen.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactElement, useState } from 'react'
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
+
+import { MicrophoneCircleIcon, ImageCircleIcon } from '../../../assets/images'
+import Button from '../../components/Button'
+import CustomTextInput from '../../components/CustomTextInput'
+import Dropdown from '../../components/Dropdown'
+import RouteWrapper from '../../components/RouteWrapper'
+import { TitleSpacing } from '../../components/Title'
+import { HintText } from '../../components/text/Hint'
+import { BUTTONS_THEME, getArticleWithLabel } from '../../constants/data'
+import { getLabels } from '../../services/helpers'
+
+const Root = styled.View`
+  padding: ${props => props.theme.spacings.md};
+  flex: 1;
+`
+
+const SaveButton = styled(Button)`
+  margin-top: ${props => props.theme.spacings.lg};
+  align-self: center;
+`
+
+const AddAudioButton = styled(Button)`
+  margin-top: ${props => props.theme.spacings.md};
+  justify-content: flex-start;
+  padding: 0;
+`
+const AddImageButton = styled(AddAudioButton)`
+  margin-bottom: 0;
+  margin-top: ${props => props.theme.spacings.sm};
+`
+
+const StyledHintText = styled(HintText)`
+  margin-left: ${props => props.theme.spacings.xxl};
+`
+
+const UserVocabularyEditScreen = (): ReactElement => {
+  const [word, setWord] = useState<string>('')
+  const [articleId, setArticleId] = useState<number | null>(null)
+  const [wordErrorMessage, setWordErrorMessage] = useState<string>('')
+  const [articleErrorMessage, setArticleErrorMessage] = useState<string>('')
```

Comment:
Optional: I am still a fan of using `null` as default value here. Makes things a lot more explicit.
```suggestion
  const [wordErrorMessage, setWordErrorMessage] = useState<string | null>(null)
  const [articleErrorMessage, setArticleErrorMessage] = useState<string | null>(null)
```

Reply:
I think in this case you could also just use booleans since it will only ever be `errorMessage` anyway.

Reply:
refactored to boolean. thx for the hint